FirebaseAppCheck/Sources/Core/APIService/FIRAppCheckAPIService.m

```diff
@@ -167,6 +167,10 @@ - (instancetype)initWithURLSession:(NSURLSession *)session
                           }];
 }
 
+- (dispatch_queue_t)defaultQueue {
+  return dispatch_get_global_queue(QOS_CLASS_DEFAULT, 0);
```

Comment:
Discussed offline to standardize the QoS if the queues this classes dispatches methods on.

I read that GCD queues that created without specifying a QoS default to the `default` priority. Considering this, would creating internal default QoS queues for `FIRAppCheckAPIService` and `FIRAppAttestAPIService` be better since the queues will have more defined labels (for debugging).